data/tribes/buildings/productionsites/amazons/shipyard/init.lua

```diff
@@ -51,6 +51,12 @@ wl.Descriptions():new_productionsite_type {
          -- TRANSLATORS: Completed/Skipped/Did not start working because ...
          descname = _("working"),
          actions = {
+            -- steps from building to ship: 2-9
+            -- worker time: 5.5 sec
+            -- number of wares to carry: 5 + 5 + 3 + 3 = 16
+            -- (see data/tribes/immovables/shipconstruction_amazons/init.lua)
+            -- min. time total: 5 + 16 * (2 * 2 * 1.8 + 5.5 + 20 + 35) = 1088.2 sec
```

Comment:
where do the 5s at the beginning come from.
the 5 seconds in the ship_construction immovable program are only used for drawing the immovable and do not block anything